### ISSUE TITLE
[Accessibility Bug] Accessibility-->Big Black Cursor -->Scroll Up/Down is not working

### ISSUE THREAD

**Issue by ChindhuB** [topcoder-platform/Training-Challenges#190]
Steps to Reproduce: Step 1: Go to https://www.topcoder.com
Step 2: Click on Accessibility icon
Step 3: Click on Big Black Cursor
Step 4: Try to scroll up or down using scroller 

Actual Result: Not able scroll up/down
Expected Result: Scroll up/down needs to be working fine.

Device/OS/Browser Information: Desktop/Windows/(Chrome -Version 87.0.4280.141(64-bit))

Screenshots / screencast URL:

https://user-images.githubusercontent.com/46881794/105827620-368be800-5f90-11eb-99af-f0cf4e72bc8d.mp4
